init.lua

```diff
@@ -32,7 +32,7 @@ if not minetest.registered_privileges[areas.config.self_protection_privilege] th
 	})
 end
 
-if minetest.settings:get_bool("log_mod") then
+if minetest.settings:get_bool("log_mods") then
```

Comment:
Needs documentation (somewhen)

Reply:
Should I resolve this conversation?

Reply:
Was just a note. If you find more stuff to fix in areas, you could take the opportunity to document this setting. Minor issue anyway.